main.css

```
@@ -0,0 +1,928 @@
+@font-face {
+    font-family: Asap;
+    src: url('./assets/fonts/Asap-Regular.otf')
+}
+@font-face {
+    src: url("./assets/fonts/segoe-wp-light.ttf");
+    font-family: "Segoe WP";
+}
+@font-face {
+    src: url("./assets/fonts/PTS55F.ttf");
+    font-family: "PT Sans";
+}
+@font-face {
+    src: url("./assets/fonts/OpenSans-Regular.ttf");
+    font-family: "Open Sans";
+}
+@font-face {
+    font-family: "PT Sans Narrow";
+    src: url("./assets/fonts/PTN57F.ttf")
+}
+@font-face {
+    font-family: "Lucida Sans";
+    src: url("./assets/fonts/lucida-sans/LSANSI.TTF")
+}
+
+body {
+    width: 1300px; 
+    margin: 0 auto;
+}
+header {    
+    background-image: 
+      url('./assets/images/Layer\ 20.png'),
+      url('./assets/images/croppedL19.png'),
+      url('./assets/images/Layer\ 1.png');
+    background-repeat: no-repeat;
+    height: 566px;
+    text-transform: uppercase; 
+}
+.banner {
+    display: flex;
+    line-height: 85px;
+}
+.logo-banner {   
+    background-image: url("./assets/images/Logo banner.svg");
+    background-repeat: no-repeat;
+    background-position-y: 27px;
+    margin: 0;
+    display: flex;
+    margin-left: 140px;
+    /* margin-top: 28px; */
+    line-height: 85px;
+}
+h1 {
+    margin: 31px 87px;
+    font-family: "Open Sans";
+    font-size: 41px;
+    letter-spacing: 1.5px;
+    color: #ffffff;
+    font-style: italic;
+    font-weight: lighter;
+}
+nav {
+    display: flex;
+    margin-left: 166px;
+    margin-top: 56px;    
+}
+
+nav ul {
+    padding-inline-start: 20px;
+    display: flex;
+    list-style-type: none;
+    width: 433px;
+    height: 41px;
+    line-height: 41px;
+    background-image: linear-gradient(0deg, #f5c505 0%, #f5c505 100%);
+    box-shadow: 
+        -10px -8px  rgba(220, 176, 5, 0.27),
+        -5px -4px  rgb(220, 176, 5, 0.27) ;
```

you used `rgb` but passed 4 parameters
```suggestion
        -5px -4px  rgba(220, 176, 5, 0.27) ;
```